package.xml

```diff
@@ -10,6 +10,8 @@
 
   <build_depend>catkin</build_depend>
   <build_depend>python</build_depend>
+  <build_depend version_gte="0.4.0">python-catkin-tools</build_depend>
+  <run_depend version_gte="0.4.0">python-catkin-tools</run_depend>
```

Comment:
If I remember it correctly, the version are ignored by rosdep and are just for the release packaging.
However, they are good indicators for people that are debugging problems.